### PR TITLE
added folding bikes from tcgm_bikebackpack

### PR DESCRIPTION
**Pull request by copyosaurus**
## What type of PR is this.
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement

### What have you changed and why?
Information:

- added all bikes to civ vehicle array so you can be undercover in any bike
- disabled spawning of military variants (the once that actually fold into backpacks) for civs
- added military variants to the light vehicle arrays of nato and csat

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in singleplayer?
2. [ ] Have you loaded the mission in LAN host?
3. [x] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
- look for spawned bikes :)
********************************************************
Notes:
I actually only checked civ spawning and undercover yet. Didn't have a look for NATO or CSAT spawning with bikes.

### CODE REVIEW

**Comment by Bob-Murphy** [comment]
Look @SonorousMeerkat , somebody did a thing!

**Comment by SonorousMeerkat** [comment]
Can more than one person ride these bikes?

**Comment by Lazejun** [comment]
Nope 1 Seat and Adding it to Csat and NatoLight which USE Default Crew leads to Nato and CSAT Units spawning in any other Modset, also the issue of a QRF of 1 Bike with 1 Guy per "Vehicle" if Nato/CSATLight is choosen

**Comment by Bob-Murphy** [comment]
In my opinion it only would make sense to add them as civilian vehicles.
I don't see a reasonable argument why we should add them as "patrolvehicles" or such.
With the new UI we also could make them buyable when available.

**Comment by SonorousMeerkat** [comment]
Agreed

File: A3-Antistasi/Templates/AddonVics/tcgm_Civ.sqf [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+			,"TCGM_Bike_hex",0.0    // Folding Bike (Hex Green)
+			,"TCGM_Bike_hexa",0.0   // Folding Bike (Hex Arid)
+			,"TCGM_Bike_hexw",0.0   // Folding Bike (Hex Winter)
+			,"TCGM_Bike_blk",2.0    // Folding Bike (Black)
+			,"TCGM_Bike_blu",2.0    // Folding Bike (Blue)
+			,"TCGM_Bike_wht",2.0    // Folding Bike (White)
+			,"TCGM_Bike_red",2.0    // Folding Bike (Red)
+];
+
+vehNATOLight append [
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/AddonVics/tcgm_Civ.sqf:15]: Remove these appends. The bike are unsuitable for these arrays will certainly cause problems.

File: A3-Antistasi/Templates/AddonVics/tcgm_Civ.sqf [hunk 2]
```diff
@@ -0,0 +1,26 @@
+civVehCommonData append [
+            "TCGM_Bike_oli",0.0     // Folding Bike (Olive)
+			,"TCGM_Bike_snd",0.0    // Folding Bike (Sand)
+			,"TCGM_Bike_nht",0.0    // Folding Bike (Black)
+			,"TCGM_Bike_win",0.0    // Folding Bike (White)
+			,"TCGM_Bike_hex",0.0    // Folding Bike (Hex Green)
+			,"TCGM_Bike_hexa",0.0   // Folding Bike (Hex Arid)
+			,"TCGM_Bike_hexw",0.0   // Folding Bike (Hex Winter)
+			,"TCGM_Bike_blk",2.0    // Folding Bike (Black)
```
Comment (SonorousMeerkat) [inline A3-Antistasi/Templates/AddonVics/tcgm_Civ.sqf:9]: I'm not sure on this weighting. What is the rough proportion of bikes:cars?

I think 1.0 would be plenty no?

**Comment by copyosaurus** [comment]
I think bike cops would be a fun addition ...

And the idea of Nato patrolling on bikes seemed funny to me. But if it causes problems with crews that are too large I'll sadly never see that.

Buying would be nice but I couldn't figure out how to add a button. Would be nice for early game recon and it doesn't suck as much as the quads.

**Comment by SonorousMeerkat** [comment]
Unfortunately all our vehicles need more than 1 seat for them to be used, and they *need* to be side correct for the AI right now as they spawn with the normal vehicle crew 90% of the time.

**Comment by Bob-Murphy** [comment]
As the patrols don't only follow the roads but also go offroad and then could very easily be shot, it's a no.
Of course it would be fun to watch, but it wouldn't make sense.
Buying, as I wrote above, will be added with the UI overhaul as this has better options implementing it.

**Pull request merged**